### PR TITLE
RadarGun 2.1.0.Final

### PR DESCRIPTION
**Pull request by mcimbora**
Preparing for 2.1.0 release
- Upgrading version to 2.1.0.Final
- Removed old schema versions
- Replaced schema symlinks by xsd files

### CODE REVIEW

**Comment by alanfx** [comment]
Looks good besides the one missing schema file

**Comment by mcimbora** [comment]
@alanfx Thanks for the review. The schema is embedded in xsd files of plugins, which use jcache module as a dependency, e.g. jdg65, infinispan70, hazelcast3, which reminds me of the fact that infinispan71, infinispan72 and infinispan80 should use jcache dependency as well. I'll issue a PR with fix.

**Comment by mcimbora** [comment]
Hmm didn't realize they use infinispan70 as a dependency, this solves the issue and no PR is necessary :-)

File: core/src/main/resources/benchmark-jcache.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<!-- RadarGun 2.0 benchmark -->
-<benchmark xmlns="urn:radargun:benchmark:2.0">
+<!-- RadarGun 2.1 benchmark -->
+<benchmark xmlns="urn:radargun:benchmark:2.1">
```
Comment (alanfx) [inline core/src/main/resources/benchmark-jcache.xml]: So this schema reference works based on the schema being embedded in the main plugin schema?

**Comment by alanfx** [comment]
Integrated. Thanks @mcimbora! On to RadarGun 3.0!!!

**Comment by rvansa** [comment]
Guys, release commit should not go into master branch! The commit should be just tagged with 2.1.0.Final. Then, master should be updated with another commit bumping 2.1.0-SNAPSHOT to 3.0.0-SNAPSHOT.

**Comment by rvansa** [comment]
@mcimbora will sort it out, no more pushing into master until he starts 3.0.0-SNAPSHOT there, please.

**Comment by alanfx** [comment]
Sorry, @rvansa I guess we need a document for the release process?

**Comment by rvansa** [comment]
@alanfx Yes, it should... I've just used +- the same process as Infinispan does.

**Comment by alanfx** [comment]
@rvansa ok, I haven't done one of those releases either, so I apologize for my ignorance!

**Comment by mcimbora** [comment]
@alanfx @rvansa I think we might want to introduce a new 2.x branch for future fixes. Wdyt?

**Comment by rvansa** [comment]
@mcimbora sure, although I doubt there will be much fixes...

**Comment by alanfx** [comment]
@mcimbora I think it is worth having a branch though, just in case

**Pull request closed without merging**